src/components/tutorials/advanced-tutorial.jsx

```diff
@@ -50,7 +50,7 @@ class AdvancedTutorial extends React.Component {
           <p className="ds-c-alert--text">Looks like the screen is too narrow to show you the API payloads alongside the tutorial. As you see references to code on the right, you'll be able to find it all the way at the bottom. Clicking the buttons to progress through the tutorial will update the code!</p>
         </div>
       </div>
-      <div className="container">
+      <div>
```

Comment:
Extra `<div>` here and in basic-tutorial are required to wrap as a single component